frontend-svelte/src/routes/new_challenge/[slug]/+page.svelte

```diff
@@ -1,146 +1,38 @@
 <script>
-    import { onMount } from "svelte";
```

Comment:
You moved the logic to `new_challenge/[slug]/+page.svelte` but did not delete `components/CodeMirror.svelte`. Any specific reason for this?